src/styles/cart.scss

```diff
@@ -0,0 +1,123 @@
+#cart section:nth-child(2){padding:0;}
+.cart-header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: #000;
+    h2{color:#fff;}
```

Comment:
use variables

Reply:
Used a variable instead of giving color code everywhere in cart.scss